<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="startPushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closePushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="infoTextBrowser"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Difficulty:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserDifficulty">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberScore">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>401</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Elapsed time:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>403</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 11pt &quot;Clean&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>403</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minute(s)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>434</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 0, 0);
font: 11pt &quot;Clean&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>434</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Second(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHard">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>511</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNormal">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>511</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEasy">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>511</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
